<launch>
    <node pkg="rsd_mes_client" name="mes_client" type="rsd_mes_mobile_client_node.py" output="screen" >
	<param name="mes_command_topic" value="mes_mobile_command" />
	<param name="mes_status_topic" value="mes_mobile_status" />
	<param name="update_duration" value="1" /> <!-- Unit: Seconds -->
	<param name="server_address" value="http://192.168.1.50:8000" />
	<param name="robot_id" value="1" />
    </node>
</launch>
